Sources/RTMP/RTMPConnection.swift

```diff
@@ -3,6 +3,8 @@ import Foundation
 
 /// The RTMPResponder class provides to use handle RTMPConnection's callback.
 open class RTMPResponder {
+    static let empty = RTMPResponder(result: { _ in })
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/shogo4405/HaishinKit.swift/blob/d7d691952432638df6a0445db4675b1cfb127eb9/Sources/RTMP/RTMPConnection.swift#L6"></span>The 'stat' is the require reason api. Please check privacy manifest files.</td>
    </tr>
  </tbody>
</table>